client/src/states/loading.rs

```diff
@@ -30,7 +31,7 @@ pub struct LoadingState {
     load_ui: Option<Entity>,
 }
 
-impl SimpleState for LoadingState {
+impl<'a, 'b> State<GameData<'a, 'b>, ExtendedStateEvent> for LoadingState {
     fn on_start(&mut self, data: StateData<'_, GameData<'_, '_>>) {
         load_configs(data.world);
         self.load_ui = Some(data.world.exec(|mut creator: UiCreator<'_>| {
```

Comment:
I guess it's not possible to use this way of loading resources, or to do it in a different way.